cumulus_etl/loaders/fhir/export_log.py

```diff
@@ -237,13 +237,37 @@ def status_progress(self, response: httpx.Response):
 
     def status_complete(self, response: httpx.Response):
```

Comment:
These changes are informed by the wiki page for this log format: https://github.com/smart-on-fhir/bulk-data-client/wiki/Bulk-Data-Export-Log-Items/

And specifically, the recent changes to support a new feature of bulk exporting: partial manifests (which let you get some export results before the whole export has finished). The ETL client doesn't support this yet, so this logging support is bare bones. But it adds the newly expected events to the log.